internal/services/aws_reservation_service.go

```diff
@@ -48,10 +48,8 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 	reservation.Status = "Created"
 	reservation.Provider = models.ProviderTypeAWS
 	reservation.Steps = 2
-	if payload.Name != nil {
-		newName := config.Application.InstancePrefix + *payload.Name
-		reservation.Detail.Name = &newName
-	}
+	newName := config.Application.InstancePrefix + payload.Name
+	reservation.Detail.Name = &newName
```

Comment:
Will this behave correctly when there is an empty name?
Before we had nil for name not set, now we have empty string.
I don't know if that matters for AWS, but I'd rather be sure (our IQE pipeline does not test this as it is setting the name).

Reply:
What you mean? I believe that `"" + "" = ""`.

Reply:
oki make sense :) I was talking about what we send down the road :)